go.mod

```diff
@@ -127,13 +135,13 @@ require (
 	github.com/hashicorp/consul/api v1.24.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru/v2 v2.0.6
-	github.com/hectane/go-acl v0.0.0-20190604041725-da78bae5fc95
+	github.com/hectane/go-acl v0.0.0-20230122075934-ca0b05cb1adb // indirect
```

Comment:
We should avoid unnecessarily bumping dependencies like this one (it's a pretty big bump!). To do that, you should change the version in the `pkg/util/filesystem/go.mod` file to match that of this `go.mod` file on the main branch

Reply:
good shout. Reverted the library upgrade